tests/SoldiersHealth.test.ts

```diff
@@ -0,0 +1,10 @@
+import request from 'supertest';
+import router from '../src/routes/routes';
+
+const app = router;
+
+describe('Is Soldier Alive', () => {
+  it('Should Send an API Get Request If Soldier Is Alive, Expected 200:', () => {
```

Comment:
should return 'I am alive'

I don't see what the soldier has to do with this. The health check is to see if the **server** is alive